contract_comm/validators/validators.go

```diff
@@ -148,7 +144,6 @@ func GetValidatorData(header *types.Header, state vm.StateDB, validatorAddresses
 	var validatorData []istanbul.ValidatorData
 	for _, addr := range validatorAddresses {
 		validator := struct {
-			Name           string
```

Comment:
Shouldn't URL be gone too?

Reply:
Indeed, it was removed from the ABI by someone else before already